.github/ISSUE_TEMPLATE/new_component_template.md

```diff
@@ -4,14 +4,40 @@ about: Nieuw component in LUX Design System
 ---
 
 ## Omschrijving
-<!-- 
+
+<!--
 Beschrijf het component en als er een community component gebruik wordt als basis welke wijzigingen er zijn.
 -->
 
 ### Definition of done
-<!-- Wijziging of uitbreiding op de D.O.D. -->
+
+#### Component
+
+- [ ] Component is beschikbaar in de index
+- [ ] Design tokens worden gebruikt
+
+#### Storybook
+
+- [ ] Playground story
+- [ ] States en properties stories
+- [ ] Korte beschrijving van het component
+  - Wat doet het
+  - Basis van het component (bijv Utrecht)
+  - Aanpassingen en/of uitbreidingen
+  - Relatie met andere componenten (bijv. Form field label en Form field)
+
+#### Tests
+
+- [ ] Het component rendert
+- [ ] Aanpassingen en/of uitbreidingen werken
```

Comment:
Is dit die beschrijving op Utrecht Storybook? Dan graag de link er bij zetten

Reply:
Done